packages/dev-tools-pages/ts/components/ContentBlock.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { ContextInterface } from 'ts/context';
+import { media } from 'ts/variables';
+
+import { Alpha, Beta } from './Typography';
+
+const Base = styled.div`
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    :not(:last-of-type) {
+        margin-bottom: 6.25rem;
+    }
+    ${Beta} {
+        margin-bottom: 2.5rem;
+    }
+    ${media.small`
+        display: block;
+        :not(:last-of-type) {
+            margin-bottom: 3.125rem;
+        }
+    `};
+`;
+
+const Content = styled.div`
+    width: 66.693548387%;
+    ${media.small`
+        width: 100%;
+    `};
+`;
+
+const Item = styled.div`
+    p {
+        max-width: 31.25rem;
+    }
+
+    &:not(:last-of-type) {
+        margin-bottom: 2.5rem;
+        ${media.small`
+            margin-bottom: 1.875rem;
+        `};
+    }
+`;
+
+const StyledTitle = styled(Alpha)`
+    color: ${props => props.color};
+    ${media.small`
+        & + div {
+            margin-top: 1.5rem;
+        }
+    `};
+`;
+
+interface ContentBlockProps extends ContextInterface {
+    title: string;
+    main?: boolean;
+    children?: React.ReactNode;
+}
+
+const ContentBlock: React.StatelessComponent<ContentBlockProps> = props => {
+    const children = React.Children.map(props.children, child => {
+        return <Item>{child}</Item>;
+    });
+
+    const Title = props.main ? StyledTitle : Beta;
+
+    return (
+        <Base>
+            <Title color={props.colors}>{props.title}</Title>
+            {children ? <Content>{children}</Content> : null}
```

Comment:
Rather then just do a `falsey` check here, can we do `_.isUndefined(children)`? It's stricter and therefore less brittle.

Reply:
Since I'm not using lodash on any of the sites I did `children === undefined` instead. Is that ok?

Reply:
I'd strongly prefer you use Lodash here and for the 0x.org site since we use it across our entire codebase. We want to be able to `find-and-replace` it in the future, if we plan to move to something else.